.pipelines/templates/.builder-release-template.yaml

```diff
@@ -17,6 +17,25 @@ steps:
       vstsFeedPackage: asc-baseline
       vstsPackageVersion: '1.1.0-268'
       downloadDirectory: vhdbuilder/packer
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Kata CC UVM artifact'
+    inputs:
+      source: specific
+      project: CloudNativeCompute
+      pipeline: 318279
+      specificBuildWithTriggering: true
+      buildVersionToDownload: 'latest'
+      downloadType: 'single'
+      artifactName: 'drop_container_job_sign_uvm'
+      itemPattern: |
+        **/kata-containers-igvm-debug.img**
+        **/igvm-measurement**
+        **/kata-containers-igvm.img**
+        **/kata-containers.img**
+        **/kata-containers-initrd-base.img**
+        **/reference-info-base64**
+      downloadPath: $(Build.SourcesDirectory)
+    condition: eq(variables.OS_SKU, 'CBLMariner')
```

Comment:
should we only do this when we're building kata distros specifically?

Reply:
That would require us to have a separate `vhd-image-builder-mariner-kata.json` as we need to move the downloaded artifact through provisioners and it doesn't take if conditions.

The tradeoff here is to:
- Either have a separate `vhd-image-builder-mariner-kata.json` , which we kinda want to avoid.
- Or Download uvm artifacts for all mariner builds, which we thought to be better as it's ~300MB on the building machine.